pkgs/tools/misc/android-tools/default.nix

```diff
@@ -38,23 +39,33 @@ stdenv.mkDerivation rec {
     perl
     go
   ];
+
   buildInputs = [
     protobuf
     zlib
     gtest
     brotli
     lz4
     zstd
-    libusb1
     pcre2
+    fmt
+    udev
   ];
+
   propagatedBuildInputs = [ pythonEnv ];
 
   preConfigure = ''
     export GOCACHE=$TMPDIR/go-cache
   '';
 
-  meta = with lib; {
+  # android-tools uses libusb API that has not been released yet
+  # use newer bundled libusb until the new libusb release arrive
```

Comment:
Can we include a link to relevant upstream (whether android-tools or libusb) discussions here? And if it's known yet what version of libusb will ship the relevant API, calling out that specific version here might be useful.